sys/linux/sys.txt

```diff
@@ -359,6 +359,8 @@ sysfs$2(option const[2], fsindex intptr, fsname buffer[out])
 sysfs$3(option const[3])
 statfs(path ptr[in, filename], buf buffer[out])
 fstatfs(fd fd, buf buffer[out])
+fsinfo$path(path ptr[in, filename], buf buffer[out])
```

Comment:
This is missing a lot.  The actual prototype is different and more complex:

```
SYSCALL_DEFINE6(fsinfo,
                int, dfd,
                const char __user *, pathname,
                const struct fsinfo_params __user *, params,
                size_t, params_size,
                void __user *, result_buffer,
                size_t, result_buf_size)
```

Take a look at ``include/uapi/linux/fsinfo.h`` and ``Documentation/filesystems/fsinfo.rst``.

Reply:
Thanks. I think I looked at the wrong document when writing the description. I've added the description correspondingly and pushed.